src/components/HomeBanner/homeBanner.scss

```diff
@@ -0,0 +1,125 @@
+@import "@styles/variables";
+
+.banner {
+  border-radius: 0.5rem;
+  position: relative;
+
+  &__image {
+    border-radius: 0.5rem;
+    height: 30rem;
+    margin-bottom: 5rem;
+    width: 100%;
+  }
+
+  &__auth-info-wrapper {
+    color: $primary-color;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    gap: 1rem;
+    padding: 2rem;
+    position: absolute;
+    top: 14rem;
+    width: 40%;
```

Comment:
remove?

Reply:
I need this for the after-login view according to the Figma file design. 